Custom-Audio-Driver/Custom-Audio-Driver/DefaultAudioDevice.swift

```diff
@@ -655,11 +659,11 @@ func updateRecordingDelay(withAudioDevice audioDevice: DefaultAudioDevice) {
         let session = AVAudioSession.sharedInstance()
         let interval = session.inputLatency
         
-        audioDevice.recordingDelayHWAndOS += UInt32(interval * 1000000)
+        audioDevice.recordingDelayHWAndOS += UInt32(interval * DefaultAudioDevice.kToMicroSecond)
         let ioInterval = session.ioBufferDuration
         
-        audioDevice.recordingDelayHWAndOS += UInt32(ioInterval * 1000000)
-        audioDevice.recordingDelayHWAndOS += UInt32(audioDevice.recordingAudioUnitPropertyLatency * 1000000)
+        audioDevice.recordingDelayHWAndOS += UInt32(ioInterval * DefaultAudioDevice.kToMicroSecond)
```

Comment:
can we call this "recordingDelay" in line with "playoutDelay" ?

Reply:
They are different things. This particular change it was only regarding moving the 1000000 into a variable. Nothing else was changed here

Reply:
I meant the var name